src/app/components/splash-screen/splash-screen.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="app-splash-screen"
+[ngStyle] ="{'left': windowWidth}" >
+    <div class="app-splash-screen-inner">
+        <div class="app-logo"><img alt={{img_text}} src="../../../assets/icons/logo.svg"></div>
+        <div class="app-label">Todos</div>
```

Comment:
https://github.com/sza-wtag/ToDoApp/pull/1/files#r1228077425

Reply:
declared imageUrl property for the purpose.